docs/library/datadisplay/list.md

```diff
@@ -1,8 +1,8 @@
 ---
 components:
-    - rx.radix.themes.list.item
-    - rx.radix.themes.list.ordered
-    - rx.radix.themes.list.unordered
+    - rx.radix.list.item
```

Comment:
Can we event just do `rx.list.item` here?

Reply:
Yes, we could. I left them in the radix namespace so its explicit that it's a radix component. But I could change them to use the `rx.` namespace if it looks problematic

Reply:
We can do in a follow up if needed